src/models/event.ts

```diff
@@ -1657,6 +1657,17 @@ export class MatrixEvent extends TypedEventEmitter<MatrixEventEmittedEvents, Mat
     public setThreadId(threadId?: string): void {
         this.threadId = threadId;
     }
+
+    /**
+     * True if:
+     *  - The event type is {@link EventType.RoomMessage}
+     *  - The `msgType` of the event content is {@link MsgType.KeyVerificationRequest}
+     *
+     * @returns true if the event is key verification request
+     */
+    public isKeyVerificationRequest(): boolean {
+        return this.getType() === EventType.RoomMessage && this.getContent().msgtype === MsgType.KeyVerificationRequest;
+    }
```

Comment:
I'd suggest you inline this. Or make it a utility function in `rust-crypto/index.ts`. I don't really think it's useful enough to form part of the public API for `Event`.